statsmodels/duration/hazard_regression.py

```diff
@@ -1347,26 +1347,20 @@ def get_distribution(self, params):
 
         Parameters
         ----------
-        params : arrayh-like
+        params : array-like
             The model proportional hazards model parameters.
 
         Returns
         -------
-        A list of objects of type scipy.stats.distributions.rv_discrete
+        See statsmodels.duration.hazard_regression.rv_discrete_float
```

Comment:
this should not be a "see", we don't want to have to look somewhere else to find the explanation for the returns